packages/pigeon/CHANGELOG.md

```diff
@@ -1,13 +1,26 @@
+## 12.0.0
+
+* Adds error handling on Flutter API methods.
+* **Breaking Change** [kotlin] Flutter API methods now return `Result<return-type>`.
+* **Breaking Change** [swift] Flutter API methods now return `Result<return-type, FlutterError>`.
+* **Breaking Change** [java] Removes `Reply` class from all method returns and replaces it with `Result`.
+  Changes required: Replace all `Reply` callbacks with `Result` classes that contain both `success` and `failure` methods.
```

Comment:
If there are intended to be formatted as nested, as they are in the raw text, I believe they will need to be sub-bullets.